lib/beaker/dsl/install_utils.rb

```diff
@@ -1279,6 +1284,10 @@ def install_puppetagent_dev_repo( host, opts )
         when /^(debian|ubuntu|cumulus)$/
           on host, "dpkg -i --force-all #{onhost_copied_file}"
           on host, "apt-get update"
+        when /^windows$/
+          result = on host, "echo #{onhost_copied_file}"
+          onhost_copied_file = result.raw_output.chomp
+          on host, "cmd /C 'start /w #{onhost_copied_file}'"
```

Comment:
I'm also not sure about the need for single quotes, we are not currently doing that in puppet: https://github.com/puppetlabs/puppet/blob/master/acceptance/lib/puppet/acceptance/install_utils.rb#L252